env/build-requirements.txt

```diff
@@ -6,3 +6,4 @@ wheel
 setuptools
 black
 pre-commit
+onnx
```

Comment:
Do we need this?

Reply:
onnx is a dependency for onnx-mlir project

Reply:
It's not listed as a dependency in their project requirements.  They really require python to run their optimizer driver?

Reply:
This is something I found by building it that it requires onnx at least for Mac OSX build, I suspect they forgot to mention in the project requirements.